apps/daffio/src/app/core/sidebar/containers/sidebar-viewport/sidebar-viewport.component.ts

```diff
@@ -5,6 +5,9 @@ import { Observable } from 'rxjs';
 import { CloseSidebar, OpenSidebar, SetSidebarState } from '../../actions/sidebar.actions';
 import * as fromDaffioSidebar from '../../reducers/index';
 import { DaffSidebarMode } from '@daffodil/design';
+import { DaffioGuideList } from 'apps/daffio/src/app/docs/shared/models/guide-list';
```

Comment:
These paths seem evil.

Reply:
Addressed in 5de79c050489de6e9ea9890dcbd9152c51a22c04